src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,88 @@
+REM
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+REM %3: Response files location
+
+@echo off
+setlocal
+
+set BridgeHost=
+pushd %2
+pushd bin\wcf\tools\Bridge
+if [%ERRORLEVEL%]==[0] (
+    call Bridge.exe -stop
+    popd
```

Comment:
This is the same logic as stopbridge.cmd except I only pop if push has been successful(push could be fail if the tools\Bridge has not been built yet.)

We will need to change to use IIS/Self Hosted service once it's ready.  I'd suggest we don't spend more time on this PR in the meat time. I will cancel the PR if no objection.